src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
I tried to use the [reset function](https://github.com/gitpoint/git-point/blob/master/src/utils/method-helpers.js#L20) (defined in `utils/method-helpers.js`) but I had an error, something like:
> 'Issue' key is not defined, you should use Login, Main ...

I'll try now to use directly `navigation.reset` 😃 

Thanks for reviewed this!